dev/workflow/setup_expt.py

```diff
@@ -236,7 +240,7 @@ def _common_args(parser):
         parser.add_argument('--idate', help='starting date of experiment, initial conditions must exist!',
                             required=True, type=lambda dd: to_datetime(dd))
         parser.add_argument('--edate', help='end date experiment', required=False, type=lambda dd: to_datetime(dd))
-        parser.add_argument('--account', help='HPC account to use; default is host-dependent', required=False, default=os.getenv('HPC_ACCOUNT'))
```

Comment:
I think I like retaining the ability for users to specify their account at runtime.  There are situations where some individuals have multiple billable accounts and may need experiment A running on one account and experiment B on another (e.g. I used to bill to dras-aida for AI-based DA and nesdis-rdo1/2 for OSE work).